presto-hive-metastore/src/main/java/com/facebook/presto/hive/MetastoreClientConfig.java

```diff
@@ -194,4 +197,30 @@ public MetastoreClientConfig setRequireHadoopNative(boolean requireHadoopNative)
         this.requireHadoopNative = requireHadoopNative;
         return this;
     }
+
+    public boolean isMetastoreImpersonationEnabled()
+    {
+        return metastoreImpersonationEnabled;
+    }
+
+    @Config("hive.metastore.impersonation.enabled")
```

Comment:
hive.metastore-impersonation.enabled

Reply:
Suggest to keep this, because we have similar config "hive.hdfs.impersonation.enabled"